src/lib/addons.js

```diff
@@ -1,29 +1,28 @@
-'use strict';
+import swagger from 'swagger-stats';
+import helmet from 'helmet';
+import cors from 'cors';
+// eslint-disable-next-line prettier/prettier
+import expressShortCircuit from 'express-favicon-short-circuit';
+import spec from '../api-spec/openapi.json' assert { type: 'json' };
```

Comment:
well lookie here who's using json modules which is experimental, i dont have anything against it btw

Reply:
it's not experimental in 18. `assert` is the 18-native way, 20+ uses `with`

Reply:
oh, i'm totally wrong, it's stability 1 in node 18, same as `node:module` imports

Reply:
i thought a while back that it was finally stabilized 😢